api/schema_system.go

```diff
@@ -117,3 +117,12 @@ func UnmarshalNamingRulePayloadFormat(payload string) (*regexp.Regexp, error) {
 	}
 	return format, nil
 }
+
+// UnmarshalRequiredColumnRulePayload will unmarshal payload to RequiredColumnRulePayload.
+func UnmarshalRequiredColumnRulePayload(payload string) (*RequiredColumnRulePayload, error) {
```

Comment:
Let's unify the return value for unmarshal payload functions. We can make them all return the payload struct, or the value we need for the advisor.
PS, should check the length of `ColumnList`